include/boost/graph/isomorphism.hpp

```diff
@@ -39,8 +39,8 @@ namespace detail
         typedef typename graph_traits< Graph2 >::vertex_descriptor vertex2_t;
         typedef typename graph_traits< Graph1 >::edge_descriptor edge1_t;
         typedef typename graph_traits< Graph1 >::vertices_size_type size_type;
-        typedef typename Invariant1::result_type invar1_value;
-        typedef typename Invariant2::result_type invar2_value;
+        typedef typename Invariant1::result_type invariant_t;
+        typedef unordered_map< invariant_t, size_type > multiplicity_map;
```

Comment:
It's important to let the user choose this type, so it needs to become a template parameter with a default value, like:
```
    template < typename Graph1, typename Graph2, typename IsoMapping,
        typename Invariant1, typename Invariant2, typename IndexMap1,
        typename IndexMap2, typename InvariantCountMap = boost::unordered_flat_map< invariant_t, size_type > >
```
Note that I've used `boost::unordered_flat_map`, which generally blows `boost::unordered_map` out of the water. (See [here](https://www.boost.org/doc/libs/develop/libs/unordered/doc/html/unordered.html).)

Reply:
This is the only change left, then it's good to merge!